lib/routers/establishment/retrospective-assessments.js

```diff
@@ -53,6 +55,28 @@ const validateAction = (req, res, next) => {
   return next(new BadRequestError());
 };
 
+const calculateRaReasons = (req, res, next) => {
+  return Promise.resolve()
+    .then(() => {
+      const { ProjectVersion } = req.models;
+      return ProjectVersion.query()
+        .where({ projectId: req.project.id, status: 'granted' })
+        .orderBy('createdAt', 'desc')
+        .then(versions => {
+          let reasons;
+          return versions.find(v => {
+            reasons = getReasons(v.data);
+            return !isEmpty(reasons);
+          }) ? reasons : {};
```

Comment:
this just returns the first set of reasons it finds, it doesn't combine them if there are different reasons for each version.

Reply:
I think that makes sense.